igvc_navigation/src/path_follower/main.cpp

```diff
@@ -22,17 +22,40 @@ SmoothControl controller;
 
 void path_callback(const nav_msgs::PathConstPtr& msg)
 {
-  ROS_INFO("Follower got path");
+  ROS_INFO_STREAM("Follower got path. Size: " << msg->poses.size());
   path = msg;
 }
 
 double get_distance(double x1, double y1, double x2, double y2)
```

Comment:
I think we could probably extract this out of this class and into the robot_state class maybe?

Reply:
I'm not sure what you mean by this. Could you clarify a bit further?

Reply:
I think he means using robot state here, then have a method called distance that does this in that implementation.

Reply:
Ok gotcha. I'll create a `RobotState` object to hold odometry information and create an instance method that returns the euclidian distance to an arbitrary point.